### DISCUSSION TITLE
Starting a SelfCAD 3D printer design for a Newt case.

### DISCUSSION THREAD

**Discussion by Biffco** [Phambili-Tech/Newt_Display#23] [Ideas]
I had to import the Eagle brd file into Altium then export a step file to use in SelfCAD.  The step file has just the board, not the components (with their extrusions), but it will let me lay out the hole pattern, etc, without calipers and calculator.

A full step file or other popular 3D design format would be helpful.  Maybe Darian is watching...

Thanks.

**Comment by darianbjohnson** [comment]
I'm always watching :)....

So - like a *.obj, *.step, or  *.stl file? Any preference? How about all three. Let me know if any of these work - [https://github.com/Phambili-Tech/Newt_Display_Hardware/tree/main/3D%20File%20Formats](https://github.com/Phambili-Tech/Newt_Display_Hardware/tree/main/3D%20File%20Formats)

Note - the Sharp screen is not part of the 3D render - so you may need to take that into account into your design. I can find specs for it if needed - just let me know.

And please - if you make something, share it with CrowdSupply. I think it will qualify for a Field Report (which I think gives you a discount on your new CrowdSuppy purchase).

**Reply by weaver3294** [reply]
I tried importing the files into fusion but they all show corrupt. Any thoughts?

**Reply by darianbjohnson** [reply]
Hmm - that's weird. I'll look into in over the next few days, and will let you know what I find (or maybe I need to replace the files).

**Reply by weaver3294** [reply]
Thank you, maybe hosting the files on Google drive or elsewhere?

**Reply by darianbjohnson** [reply]
@weaver3294 - I uploaded the files to a public location - https://phambili-pub.s3.amazonaws.com/3D+File+Formats.zip

Try this and let me know if you are still seeing them as corrupt

**Reply by weaver3294** [reply]
That worked great @darianbjohnson Thank you.
